lxd/storage_btrfs.go

```diff
@@ -2849,7 +2855,7 @@ func (s *storageBtrfs) StorageEntitySetQuota(volumeType int, size int64, data in
 	switch volumeType {
 	case storagePoolVolumeTypeContainer:
 		c = data.(container)
-		subvol = getContainerMountPoint(s.pool.Name, c.Name())
+		subvol = getContainerMountPoint("default", s.pool.Name, c.Name())
```

Comment:
Can you give me a quick argument how to decide whether to use `default` or `Project()`? :)

Reply:
You should use Project() whenever you're dealing with a resource that can be part of a project:
 - A profile
 - An image
 - A container

In this case, you're dealing with a custom storage volume, which isn't a feature that's project aware and so can only belong to the default project.

Reply:
Nice explanation.